src/cli/onbranch.go

```diff
@@ -29,6 +29,13 @@ func (c icli) buildPushDeploy(cCtx *cli.Context) error {
 	if cCtx.Bool(stopOnPushFlag) {
 		return err
 	}
+
+	manifestDigest, err := c.DockerService.GetManifestDigest()
```

Comment:
Should we move this in the Deploy method?
Digest should be used `onmain` as well as the `deploy` command

Reply:
Good point @LucaLanziani